src/conversations/Conversations.ts

```diff
@@ -182,7 +184,14 @@ export default class Conversations {
     const out: ConversationV2[] = []
     for (const response of responses) {
       try {
-        const convo = this.saveInviteResponseToConversation(response)
+        let convo = this.saveInviteResponseToConversation(response)
+
+        const initialMembers = convo.context?.metadata.initialMembers.split(',')
+
+        if (initialMembers) {
```

Comment:
If the `convo.context.metadata.initialMembers` is set to `''`, this will actually return true because Javascript was invented in a long weekend and an empty array is truthy.

Reply:
Fixed in https://github.com/xmtp/xmtp-js/pull/381/commits/ad6d5821c733216035c44971442c42dd15a9ec93.